skore/src/skore/sklearn/_comparison/report.py

```diff
@@ -50,7 +50,7 @@ class ComparisonReport(_BaseReport, DirNamesMixin):
     Attributes
     ----------
     reports_ : list of :class:`~skore.EstimatorReport` or list of
-               :class:`~skore.CrossValidationReport`
+        :class:`~skore.CrossValidationReport`
```

Comment:
I cannot comment on the line before. The issue is that it is missing a backslash

```
    reports_ : list of :class:`~skore.EstimatorReport` or list of \
        :class:`~skore.CrossValidationReport`
```

I solved it in https://github.com/probabl-ai/skore/pull/1741 but I don't mind to quickly merge this PR if it is ready before.

Reply:
Hi @glemaitre, thank you for this. well spotted. I could quickly add the backslash and push.